tests/sys/audit/Makefile

```diff
@@ -49,6 +49,7 @@ SRCS.miscellaneous+=		utils.c
 TEST_METADATA+= timeout="30"
 TEST_METADATA+= required_user="root"
 TEST_METADATA+= is_exclusive="true"
+TEST_METADATA+=	required_files="/etc/rc.d/accounting /etc/rc.d/auditd"
```

Comment:
This metadata should be set for the `acct_success` testcase only, not for the entire test program.

Reply:
Uh, actually, pretty much everything calls `setup(..)`, which prods the accounting/auditd services, so I'm 99% sure this is valid as-is :(...

Reply:
`setup` starts `auditd`, but it doesn't have anything to do with `accounting`.

Reply:
Ah... ok. I see what you meant! I'll work on fixing that.